endoreg_db/serializers/video/video_file_detail.py

```
@@ -1,24 +1,13 @@
 from rest_framework import serializers
-from endoreg_db.models import VideoFile, VideoImportMeta, LabelVideoSegment, VideoMeta, SensitiveMeta
+from endoreg_db.models import LabelVideoSegment, SensitiveMeta
```

Ruff (0.12.2)

`endoreg_db.models.LabelVideoSegment` imported but unused

Remove unused import

(F401)